Jenkinsfile

```diff
@@ -116,8 +116,9 @@ pipeline {
 
                     environment {
                         TESTBED = credentials('win-testbed')
-                        TESTBED_TEMPLATE = "Template-testbed-201804050628"
-                        CONTROLLER_TEMPLATE = "Template-CentOS-7.4-Thin"
+                        // TODO: change after creation of proper templates
+                        TESTBED_TEMPLATE = "tmpl-wintb-master"
+                        CONTROLLER_TEMPLATE = "tmpl-ctrl-master"
```

Comment:
Since we already have some testbed templates created by @jablonskim , maybe we can plug them here already? What do you think about this?

Also we would need to create separate CentOS template.